src/locales/uk/translation.json

```diff
@@ -417,6 +417,7 @@
   "viewTheRegistry": "Перегляд реєстру",
   "views": "Переглядів",
   "warning": "Попередження",
+  "websiteAvailableOnDesktopDevices": "Перепрошуємо, платформа не доступна на даному екрані.",
```

Comment:
Not clear text, how should the user understand that the platform is not available on mobile devices only?

Reply:
Change to "Sorry, this website isn`t available on mobile devices. Please open a desktop version."